data/README.md

```diff
@@ -39,18 +39,27 @@ instance, they should be in the same format as the files below.
   Contains signature (node) activity levels (as absolute values) for each
   sample in the compendium derived from the eADAGE machine learning model.
   In this file, samples are rows, and signatures are columns. This file is
-  based on the
-  `all-pseudomonas-gene-normalized_HWActivity_perGene_with_net300_100models_1_100_k=300_seed=123_ClusterByweighted_avgweight_network_ADAGE.txt`
-  file, which is now in the `data/old/` folder and was generated by
+  based on:
+  ```
+  all-pseudomonas-gene-normalized_HWActivity_perGene_with_net300_100models_1_100_k=300_seed=123_ClusterByweighted_avgweight_network_ADAGE.txt
+  ```
+
+  which is now in the `data/old/` folder and was generated by
   [this script](https://bitbucket.org/greenelab/eadage/src/tip/node_interpretation/write_HWactivity.R).
 
+  **Note:** The signatures shown on the web (click
+  [here](http://adage.greenelab.com/#/signature/search?mlmodel=1) to see a
```

Comment:
Minor quibble: could this say

```markdown
...(view [this page](http://adage.greenelab.com/#/signature/search?mlmodel=1) to
see an example)...
```

Someone sensitized me to that fact that not everyone uses a mouse anymore and it looks funny to see that written when you actually need to tap or use some assistive technology to open the link another way. Also, I think it just sounds better.

Reply:
Good point. Sure!